readme.md

```diff
@@ -168,7 +188,7 @@ If your DxGrid does not display master-detail relationships and does not use the
 [Return to the table of contents.](#thetableofcontents)
 
 
-### Change the Default "No data to display" Text
+#### Change the Default "No data to display" Text
 
 In v22.1, use the following CSS rules:
 
```

Comment:
Please make the following changes below:

Line 208:
"In v22.2, use the new CustomizeElement event to change the header cells' color instead of adding a custom CSS style:"  
Please change "header cells' color" to "header cell color".

Line 249:
"In v22.2, the DxGrid automatically focuses the first editor. If you wish to focus a different editor, wrap it into the CasadingValue component and set the CascadingValue.Name property to "FocusOnEditStart"::"  
1. Please change "wrap it into" to "wrap it in".  
2. There is an extra colon at the end. Please delete it.